pandas/tests/groupby/aggregate/test_cython.py

```diff
@@ -277,3 +277,38 @@ def test_read_only_buffer_source_agg(agg):
     expected = df.copy().groupby(["species"]).agg({"sepal_length": agg})
 
     tm.assert_equal(result, expected)
+
+
+@pytest.mark.parametrize(
```

Comment:
could use all_numeric_reductions fixture (although slightly different) doesn't include `sem` but does include `kurt` and `skew`

Reply:
`kurt` is not implemented for groupby, it seems. I mainly copied this from another test in this file.

Reply:
ok for now I guess.

Reply:
yeah can you create an issue for folks to replace these with fixtures (good first)